cypher-shell/src/integration-test/resources/test.cypher

```diff
@@ -0,0 +1 @@
+RETURN 42 AS result;
```

Comment:
I would like to see a couple more tests, e.g.:

* file with more than 1 line
* file with _some_ invalid Cypher (and some valid Cypher)
* empty file